src/banman.cpp

```diff
@@ -151,8 +137,8 @@ void BanMan::Ban(const CSubNet& sub_net, const BanReason& ban_reason, int64_t ba
     }
     if (m_client_interface) m_client_interface->BannedListChanged();
 
-    //store banlist to disk immediately if user requested ban
```

Comment:
nit: I think this change makes more sense in the previous commit, since that's where you added the assert.

Reply:
Agreed. Will fix if there are more changes requested, but I don't think this is worth breaking ACKs for.

Reply:
Done.